### PR TITLE
sql,ui: clarify/cleanup the display of internal queries

### LINKED ISSUE #35460: ui: App filter on statements page has too many $internal options
When looking at the `APP:` dropdown on the statements page, we see many distinct categories of internal queries. When a user is looking through these app names to find their own application, it can be frustrating to have to search through > 10 "apps" with the $internal prefix. 

image

Instead, we should probably group these in the UI under a single internal heading. The additional detail provided for the different internal query types is probably not helpful to most of our users.

cc @jseldess

### PR DESCRIPTION
**Pull request by knz**
Fixes  #35460.

This patch achieves two things:

- it clarifies the distinction between `ReportableAppNamePrefix` (avoids
  scrubbing in reporting, used both by internal queries and CLI
  utilities) and `InternalAppNamePrefix` (only for internal queries).
- it groups all internal queries under a single "(internal)" entry
  in the UI in the drop-down menu.

Release note (admin ui change): Queries issued internally by
CockroachDB are now displayed under a single "(internal)" application
name entry in the drop-down menu in the statements page.

### CODE REVIEW

**Comment by knz** [comment]
Example screenshot:

schermafdruk_2019-03-07_16-40-06

File: pkg/ui/src/views/statements/statementsPage.tsx [hunk 1]
```diff
@@ -181,12 +181,15 @@ export const selectStatements = createSelector(
     let statements = flattenStatementStats(state.data.statements);
     if (props.params[appAttr]) {
       let criteria = props.params[appAttr];
+	  let showInternal = false;
```
Comment (celiala) [inline pkg/ui/src/views/statements/statementsPage.tsx]: nit: looks like the indentation got shifted?

**Review by celiala — APPROVED** [review]
:lgtm: - thanks for fixing this!

File: pkg/ui/src/views/statements/statementsPage.tsx (hunk 1 above)
Comment (knz) [reply pkg/ui/src/views/statements/statementsPage.tsx]: Done.

**Pull request merged**